src/interface.js

```diff
@@ -0,0 +1,48 @@
+$(document).ready(function () {
+  var scorecard = new Scorecard
+
+  $('#new-roll').submit(function (event) {
+    event.preventDefault();
+    scorecard.roll(parseInt($("#roll").val()))
+    printScores()
+  })
+
+  function printScores() {
+    $("#scores").empty()
+    for (i = 0; i < scorecard.getNumFrames(); i++) {
+      var frame = scorecard.getFrame(i)
+      var rollDisplays = getRollDisplays(frame)
+      totalScore = scorecard.score(i + 1)
+      html = createFrameDiv(rollDisplays, totalScore)
+      $("#scores").append(html)
+    }
+  }
+
+  function getRollDisplays(frame) {
+    if (frame.getNumber() == 10) {
+      return getTenthRollDisplays(frame)
+    } else {
+      return getStandardRollDisplays(frame)
+    }
+  }
+
+  function getStandardRollDisplays(frame) {
+    return [frame.getRoll(0), frame.getRoll(1), '']
+  }
+
+  function getTenthRollDisplays(frame) {
+    if (frame.isStrike()) {
+      return [frame.getRoll(0), frame.getBonusRoll(0), frame.getBonusRoll(1)]
```

Comment:
You could refactor this with a ternary operator or two single line if statements